runtime/lua/vim/_editor.lua

```diff
@@ -670,28 +670,33 @@ local on_key_cbs = {} --- @type table<integer,function>
 ---          are applied, and {typed} is the key(s) before mappings are applied.
 ---          {typed} may be empty if {key} is produced by non-typed key(s) or by the
 ---          same typed key(s) that produced a previous {key}.
+---          If {opts.may_discard} then when {fn} returns
+---          true, {key} is ignored.
 ---          When {fn} is `nil` and {ns_id} is specified, the callback associated with
 ---          namespace {ns_id} is removed.
```

Comment:
nit: while we're here we can simplify this


```suggestion
---          When {fn} is `nil`, the callback associated with namespace {ns_id} is removed.
```